src/handlers/request/RequestHandlers.ts

```diff
@@ -0,0 +1,15 @@
+import {RequestHandler} from './RequestHandlerInterfaces.js';
+
+export default (options: RequestInit, requestHandlers?: RequestHandler[]): RequestInit => {
```

Comment:
I would change the naming of `options` to init or `requestInit` to stay consistent